registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -222,16 +222,28 @@ public MessageDTO process(MessageDTO object) {
 			}
 			else {
 			String vid = getVid(uin);
-			CredentialRequestDto credentialRequestDto = getCredentialRequestDto(vid);
 			requestWrapper.setId(env.getProperty("mosip.registration.processor.credential.request.service.id"));
-			requestWrapper.setRequest(credentialRequestDto);
+			String issuers=	env.getProperty("mosip.registration.processor.issuer");
 			DateTimeFormatter format = DateTimeFormatter.ofPattern(env.getProperty(DATETIME_PATTERN));
 			LocalDateTime localdatetime = LocalDateTime
-					.parse(DateUtils.getUTCCurrentDateTimeString(env.getProperty(DATETIME_PATTERN)), format);
+						.parse(DateUtils.getUTCCurrentDateTimeString(env.getProperty(DATETIME_PATTERN)), format);
 			requestWrapper.setRequesttime(localdatetime);
 			requestWrapper.setVersion("1.0");
-			responseWrapper = (ResponseWrapper<?>) restClientService.postApi(ApiName.CREDENTIALREQUEST, null, null,
-					requestWrapper, ResponseWrapper.class, MediaType.APPLICATION_JSON);
+			for (String key : issuers.split(SEMICOLON)) {
+				CredentialRequestDto credentialRequestDto=null;
+				String[] parts = key.split(COLON, 3);
+				credentialRequestDto = getCredentialRequestDto(regId,parts[0],parts[1],parts[2]);
+				requestWrapper.setRequest(credentialRequestDto);
+				if(parts[0].equalsIgnoreCase("mpartner-default-print")){
+					responseWrapper = (ResponseWrapper<?>) restClientService.postApi(ApiName.CREDENTIALREQUEST, null, null,
+							requestWrapper, ResponseWrapper.class, MediaType.APPLICATION_JSON);
+				}else{
+					List<String> pathsegments=new ArrayList<>();
+					pathsegments.add(regId+"#PDF");
```

Comment:
What is this #PDF suffix is about? please add a comment on the purpose of that. Also make it as a constant.

Reply:
taken care